api/client/client.go

```diff
@@ -1319,6 +1319,55 @@ func (c *Client) DeleteAllNodes(ctx context.Context, namespace string) error {
 	return trail.FromGRPC(err)
 }
 
+// StreamSessionEvents streams audit events from a given session recording.
+func (c *Client) StreamSessionEvents(ctx context.Context, sessionID string, startIndex int) (chan events.AuditEvent, chan error) {
+	request := &proto.StreamSessionEventsRequest{
+		SessionID:  sessionID,
+		StartIndex: int32(startIndex),
+	}
+
+	ch := make(chan events.AuditEvent)
+
+	stream, err := c.grpc.StreamSessionEvents(ctx, request)
+	if err != nil {
+		e := make(chan error)
+		go func() {
+			e <- trace.Wrap(err)
+			close(ch)
+		}()
+
+		return ch, e
+	}
+
+	e := make(chan error)
```

Comment:
note: the caller does not guarantee that it will ever read the channel what in turn can lead to a goroutine leak, if you make channel buffered, you will guarantee that the gorotine below will finish